vic/vic_run/src/lakes.eb.c

```diff
@@ -26,6 +26,7 @@
 
 #include <vic_def.h>
 #include <vic_run.h>
+#include <vic_driver_shared.h>
```

Comment:
This is because the get_depth(), get_volume() etc. functions are stored in a file that's part of the driver code.  I can break them out into separate files and move them to vic_run/.